test/hapi.stat.test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var assert = require('assert')
```

Comment:
Of course @AdrieanKhisbe. You're free to replace `assert` with `Code` and issue a new PR. I think this PR is about to be merged, so I think it's best if you wait until then. (or you can do it now, it's up to you really :) thanks :+1: